.gitignore

```diff
@@ -21,10 +21,9 @@ Psychtoolbox/ptbflavorinfo.txt
 .fuse_*
 liboctave.*
 liboctinterp.*
-PsychPython/__pycache__/
-PsychPython/*.pyc
-PsychPython/demos/__pycache__/
-PsychPython/demos/*.pyc
+*.pyc
+__pycache__
+*.so
```

Comment:
Ignoring *.so is not a good idea, as we have .so's shipping as part of Psychtoolbox which need to be version controlled.

Reply:
Other than that no objections. I assume this was tested on all three operating systems, so i can pull it without surprises? Note that i don't have any experience with pypi, and the bits of Python i learned last year to do the PTB Python port are slowly fading out of my memory, so until i can find the time to refresh my brain on this - which won't be soon as i'm overwhelmed with other work atm., i just mostly trust you and Jon know what you're doing and that you test properly. You get both the credit and the blame for whatever wonderful or bad happens to the Python bits atm. ;-).

python setup.py install --user is how i test if things still build and install.

Once above comment is fixed, i can pull this, and then also pull it into my own master. I assume Jon's pull request would then need some fixup with all the changed pathes. But i think Jon's pull needs some fixups anyway, for some review comments i made wrt. fillbuffer and refillbuffer and createbuffer usage, before it is mergeable.

Reply:
Pushed a commit to make it ignore more specifically:
```
PsychPython/psychtoolbox/*.so
```

> I assume this was tested on all three operating systems, so i can pull it without surprises?

I ran this on Linux x86_64, built a Python wheel on Windows 64-bit, and just tried it on macOS 64-bit and all three worked. But I agree that something like #587 would be good to automate this testing (at least that `import psychtoolbox.audio; psychtoolbox.audio.get_devices()` works) so that everyone can sleep well at night :)

> python setup.py install --user is how i test if things still build and install.

This is good if you want to be in a "permanent" state. However, if you want to edit the `.py` files and have the changes take effect the very next time you run a script then `python setup.py develop --user` (old way) or `pip install --user -ve .` (new way) is much easier because you don't have to keep doing `python setup.py install --user` commands after every change to see how things have changed\. (If you change compiled code it's more complicated, but at least any `.py` Python file changes are reflected on the next `import`.)